src/main/kotlin/model/Deck.kt

```diff
@@ -1,15 +1,15 @@
 package model
 
-class Deck {
-    companion object {
-        private val cards = CardNumber.values().flatMap { num ->
-            CardShape.values().map { shape ->
-                Card(num, shape)
-            }
-        }.shuffled().toMutableList()
+class Deck(cards: List<Card> = initCards.shuffled()) {
+    private val cards: MutableList<Card> = cards.toMutableList()
 
-        fun getCard(): Card {
-            return cards.removeAt(0)
-        }
+    fun selectCard(): Card {
+        return cards.removeAt(0)
+    }
+}
+
+val initCards: List<Card> = CardNumber.values().flatMap { num ->
```

Comment:
```suggestion
private val initCards: List<Card> = CardNumber.values().flatMap { num ->
```
Deck 클래스에서만 사용한다면, private 접근자를 붙여보면 어떨까요?

Reply:
당연히 붙였어야 했는데 놓쳤네요. 감사합니다!